pkg/auth/handler/api/workflow_get.go

```diff
@@ -12,7 +12,7 @@ import (
 func ConfigureWorkflowGetRoute(route httproute.Route) httproute.Route {
 	return route.
 		WithMethods("GET").
-		WithPathPattern("/api/workflow/v1/:instanceid")
+		WithPathPattern("/api/v1/workflows/:workflowid/instances/:instanceid")
```

Comment:
We are ignoring the workflowid in the path. If it is an input now, then we should make ensure the instance id and workflow id pair is valid.